docs/source/install/deb.rst

```diff
@@ -68,7 +68,7 @@ The format of the DEB packages is like this: <component>_<version>-<build>_amd64
 You can download the packages from this URL:
 ::
 
-    https://ops.stackstorm.net/releases/st2/0.6.1dev/debs/current/
```

Comment:
This sucks: the |version| substitution works everywhere but not in formatted text.
Can you try out this and see if it works? 
```
.. parsed-literal::

    curl -q -k -O https://ops.stackstorm.net/releases/st2/|version|/requirements.txt
```
http://stackoverflow.com/questions/8821511/substitutions-in-sphinx-code-blocks

Disclaimer: I didn't try it yet